Source/ASNetworkImageNode.mm

```diff
@@ -617,8 +660,11 @@ - (void)_lazilyLoadImageIfNecessary
         if (imageContainer != nil) {
           strongSelf->_imageLoaded = YES;
           if ([imageContainer asdk_animatedImageData] && _downloaderFlags.downloaderImplementsAnimatedImage) {
-            strongSelf.animatedImage = [_downloader animatedImageWithData:[imageContainer asdk_animatedImageData]];
+            id animatedImage = [_downloader animatedImageWithData:[imageContainer asdk_animatedImageData]];
```

Comment:
Should we unlock before calling this? Not sure the answer. Previously we've attempted to release the lock when calling the downloader (but haven't been perfect about it obviously).